MMM-MBTA.js

```diff
@@ -211,48 +320,46 @@ Module.register("MMM-MBTA", {
             this.scheduleUpdate();
         }
         
+        // Alerts
         if (this.config.showAlerts) {
-            // This might break in future updates
             var alertHeader = document.createElement("header");
-            alertHeader.className = "module-header";
+            alertHeader.className = "module-header alerts";
             alertHeader.innerHTML = "Alerts";
             wrapper.appendChild(alertHeader);
-            
+                
             var alertTable = document.createElement("table");
             alertTable.className = "small";
-            
-            for (let i = 0; i < this.alerts.length; i++) {
+
+            alertsArray = [];
+
+            for (let i = 0; i < this.stationData.length; i++) {
+                alertsArray.push(this.stationData[i].alert);
+            }
+
+            function onlyUnique(value, index, self) { 
+                return self.indexOf(value) === index;
+            }
+
+            let uniqueAlerts = [...new Set(alertsArray)];
+
+            for (let i = 0; i < uniqueAlerts.length; i++) {
+                var alertText = uniqueAlerts[i];
+                
                 var row = document.createElement("tr");
                 alertTable.appendChild(row);
                 alertTable.style.cssText = "width: inherit";
                 
-                // Icon
-                var symbolCell = document.createElement("td");
-                switch (this.alerts[i].effect_name) {
-                    case "Delay":
-                        symbolCell.className = "fa fa-exclamation-triangle";
-                        break;
-                    case "Service Change":
-                        symbolCell.className = "fa fa-exclamation-circle";
-                        break;
-                    default:
-                        symbolCell.className = "fa fa-question-circle-o";
+                var alertCell = document.createElement("td");
+                alertCell.innerHTML = alertText;
+                if (alertText === "No alerts") {
+                    alertCell.className = "light small";
+                } else {
+                    alertCell.className = "light small alert";
                 }
-                symbolCell.style.cssText = "padding-right: 10px";
-                row.appendChild(symbolCell);
-                
-                // If I ever want to add in marquee effect https://stackoverflow.com/a/21233577/5054366
-                var descCell = document.createElement("td");
-                descCell.innerHTML = this.alerts[i].header_text;
-                descCell.className = "light small";
-                row.appendChild(descCell);
-                
+                row.appendChild(alertCell);
```

Comment:
You're right, it wouldn't show anything. I insert "No alerts" because I think from a UX perspective it makes sense to inform a user who has `showAlerts` turned on (and is expecting to see something) that there's nothing to see. Otherwise they might get the impression that alerts aren't working.

I could see adding a config to not show anything. That way it's an intentional user decision.

Reply:
Yeah, if you wanted to add this feature (e.g. check if uniqueAlerts is empty, then add in "No alerts" if a config is set to true), feel free to!

Reply:
Just added!